packages/fx-core/resource/package.nls.json

```diff
@@ -998,8 +998,11 @@
   "driver.teamsApp.summary.publishTeamsAppSuccess": "Teams app %s successfully published to the admin portal.",
   "driver.teamsApp.summary.copyAppPackageSuccess": "Teams app %s was successfully copied to %s.",
   "driver.teamsApp.summary.copyIconSuccess": "%s icon(s) were successfully updated under %s.",
+  "driver.teamsApp.summary.validate": "Teams Toolkit has checked all categories of validation rules:\n\nSummary:\n%s failed, %s passed.\n%s\n%s\n\nA completed log of validations can be found in %s",
   "driver.teamsApp.validate.skip": "%s action is currently skipped, will be updated in the future version.",
   "driver.teamsApp.invalidParameter": "Following parameter is missing or invalid for %s action: %s.",
+  "driver.teamsApp.validate.invalidParameter": "Either manifestPath or appPackagePath parameter should be provided for action %s",
```

Comment:
Since the parameter name can change, can we make it as %s placeholder to avoid affecting localization?